library/bignum_core.h

```diff
@@ -17,8 +17,8 @@
  *  limitations under the License.
  */
 
-#ifndef MBEDTLS_BIGNUM_INTERNAL_H
-#define MBEDTLS_BIGNUM_INTERNAL_H
+#ifndef MBEDTLS_BIGNUM_CORE_H
```

Comment:
### Not a blocker

Not touching this file and updating those fields in the Rename Commit `#5a91c3f` and doing so in the Tidy up `#53f628a` will singificantly facilitate merging. 

The `mbedtls_mpi_core_mla` which is the only header included is removed in `#53f628a`

Reply:
once Gabor's PR goes in there this will be quite different